src/Symfony/Bundle/WebProfilerBundle/Tests/Profiler/TemplateManagerTest.php

```diff
@@ -121,13 +114,6 @@ protected function mockTwigEnvironment()
 
         return $this->twigEnvironment;
     }
-
-    protected function mockProfiler()
-    {
-        $this->profiler = $this->createMock(Profiler::class);
```

Comment:
we should remove the property then (and not use it anymore)

Reply:
Ah it is used by refernce in a test:
```php
    /**
     * if template exists in both profile and profiler then its name should be returned.
     */
    public function testGetNameValidTemplate()
    {
        $this->profiler->expects($this->any())
            ->method('has')
            ->withAnyParameters()
            ->willReturnCallback([$this, 'profilerHasCallback']);

        $this->assertEquals('FooBundle:Collector:foo.html.twig', $this->templateManager->getName(new ProfileDummy(), 'foo'));
    }
```